### ISSUE TITLE
Feature Request: Silence the dice roll sound

### ISSUE THREAD

**Issue by Drunemeton** [syl3r86/Actually-Private-Rolls#4]
Would be nice to have Actually Private Rolls silence the sound of the dice rolling so they're actually private rolls.

**Comment by peaceful-octopus** [comment]
Agreed. It might be nice to have it be optional, though, since there's something nice about recreating the at-the-table feeling of hearing the GM rolling dice behind the screen and not knowing why.

**Comment by flangelier** [comment]
#5

**Comment by flangelier** [comment]
FYI, if you want to fix it temporary, you can edit the JS manually.
Find the FoundryVTT directory (usually `appdata/local/FoundryVTT`)
Then go in `Data/modules/actually-private-rolls/` 
Then edit accutally-private-rolls.js (you can right-click the file, edit with notepad) and add this line
`	        app.data["sound"] = null;`  between 
`html.hide();` Line 15
and
`}` Line 16
